src/alchemlyb/tests/test_fep_estimators.py

```diff
@@ -1,223 +1,110 @@
-    dataset = alchemtest.namd.load_idws()
-    u_nk = namd.extract_u_nk(dataset['data']['forward'], T=300)
-
-    return u_nk
-
-def namd_idws_restarted():
-    dataset = alchemtest.namd.load_restarted()
-    u_nk = namd.extract_u_nk(dataset['data']['both'], T=300)
-
-    return u_nk
-
-def namd_idws_restarted_reversed():
-    dataset = alchemtest.namd.load_restarted_reversed()
-    u_nk = namd.extract_u_nk(dataset['data']['both'], T=300)
-
-    return u_nk
 
 
 class FEPestimatorMixin:
-    """Mixin for all FEP Estimator test classes.
-
-    """
+    """Mixin for all FEP Estimator test classes."""
 
     def compare_delta_f(self, X_delta_f):
+
         est = self.cls().fit(X_delta_f[0])
         delta_f, d_delta_f = self.get_delta_f(est)
 
         assert X_delta_f[1] == pytest.approx(delta_f, rel=1e-3)
         assert X_delta_f[2] == pytest.approx(d_delta_f, rel=1e-3)
 
     def get_delta_f(self, est):
+        # Use .iloc[0, -1] as we want to cater for both
+        # delta_f_.loc[0.0, 1.0] and delta_f_.loc[(0.0, 0.0), (0.0, 1.0)]
         return est.delta_f_.iloc[0, -1], est.d_delta_f_.iloc[0, -1]
 
 
 class TestMBAR(FEPestimatorMixin):
-    """Tests for MBAR.
+    """Tests for MBAR."""
 
-    """
     cls = MBAR
 
-    @pytest.fixture(scope="class",
-                    params=[(gmx_benzene_coul_u_nk, 3.041, 0.02088),
-                            (gmx_benzene_vdw_u_nk, -3.007, 0.04519),
-                            (gmx_expanded_ensemble_case_1, 75.923, 0.14124),
-                            (gmx_expanded_ensemble_case_2, 75.915, 0.14372),
-                            (gmx_expanded_ensemble_case_3, 76.173, 0.11345),
-                            (gmx_water_particle_with_total_energy, -11.680, 0.083655),
-                            (gmx_water_particle_with_potential_energy, -11.675, 0.083589),
-                            (gmx_water_particle_without_energy, -11.654, 0.083415),
-                            (amber_bace_example_complex_vdw, 2.41149, 0.0620658),
-                            (gomc_benzene_u_nk, -0.79994, 0.091579),
-                    ])
+    @pytest.fixture(
+        params=[
+            ("gmx_benzene_Coulomb_u_nk", 3.041, 0.02088),
+            ("gmx_benzene_VDW_u_nk", -3.007, 0.04519),
+            ("gmx_expanded_ensemble_case_1", 75.923, 0.14124),
+            ("gmx_expanded_ensemble_case_2", 75.915, 0.14372),
+            ("gmx_expanded_ensemble_case_3", 76.173, 0.11345),
+            ("gmx_water_particle_with_total_energy", -11.680, 0.083655),
+            ("gmx_water_particle_with_potential_energy", -11.675, 0.083589),
+            ("gmx_water_particle_without_energy", -11.654, 0.083415),
+            ("amber_bace_example_complex_vdw", 2.41149, 0.0620658),
+            ("gomc_benzene_u_nk", -0.79994, 0.091579),
+        ],
+    )
     def X_delta_f(self, request):
         get_unk, E, dE = request.param
-        return get_unk(), E, dE
+        return alchemlyb.concat(request.getfixturevalue(get_unk)), E, dE
```

Comment:
The usual python namespace rule applies, it will try to find the fixture locally first. If pytest cannot find it, then it will try to go up to the `conftest`.